.github/workflows/ci.yml

```diff
@@ -1,15 +1,15 @@
-name: DB test
+name: CI
 
 on: [push, pull_request]
 
 jobs:
 
-  test:
+  test_schema:
 
-    runs-on: ${{ matrix.os }}
+    name: DB schema
+    runs-on: ubuntu-18.04
     strategy:
         matrix:
-            os: [ ubuntu-18.04 ]
             pg: [ '10' ]
```

Comment:
Why does the plugin env require postgres?

Reply:
it became a single workflow now. The workflow contains 2 jobs. Only the job for testing the DB schema requires postgres/postgis.